specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+    }
+  },
+  "parameters": {
+    "Url": {
+      "name": "url",
+      "x-ms-parameter-location": "client",
+      "description": "The URL of the service account or table that is the targe of the desired operation.",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "StorageServiceProperties": {
+      "name": "storageServiceProperties",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/StorageServiceProperties"
+      },
+      "x-ms-parameter-location": "method",
+      "description": "The StorageService properties."
+    },
+    "Timeout": {
+      "name": "timeout",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "minimum": 0,
+      "x-ms-parameter-location": "method",
+      "description": "The The timeout parameter is expressed in seconds. For more information, see <a href=\"https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-queue-service-operations>Setting Timeouts for Queue Service Operations.</a>"
+    },
+    "ApiVersionParameter": {
+      "name": "x-ms-version",
+      "x-ms-parameter-location": "client",
+      "x-ms-client-name": "version",
+      "in": "header",
+      "required": true,
+      "type": "string",
+      "description": "Specifies the version of the operation to use for this request.",
+      "enum": [
+        "2018-10-10"
+      ]
+    },
+    "ClientRequestId": {
+      "name": "x-ms-client-request-id",
+      "x-ms-client-name": "requestId",
+      "in": "header",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Provides a client-generated, opaque value with a 1 KB character limit that is recorded in the analytics logs when storage analytics logging is enabled."
+    },
+    "DataServiceVersionParameter": {
+      "name": "DataServiceVersion",
+      "in": "header",
+      "required": true,
+      "type": "string",
+      "description": "Specifies the data service version",
+      "x-ms-parameter-location": "method",
+      "enum": [
+        "3.0"
+      ]
+    },
+    "TableProperties": {
+      "name": "tableProperties",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/TableProperties"
+      },
+      "x-ms-parameter-location": "method",
+      "description": "The Table properties."
+    },
+    "formatParameter": {
+      "name": "$format",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "Specifies the media type for the response",
+      "x-ms-parameter-location": "method",
+      "x-ms-parameter-grouping": {
+        "name": "QueryOptions"
+      },
+      "x-ms-client-name": "Format",
+      "enum": [
+        "application/json;odata=nometadata",
+        "application/json;odata=minimalmetadata",
+        "application/json;odata=fullmetadata"
+      ]
```

Comment:
this enum gets generated as `Enum0`. Can we add an `x-ms-enum` directive (suggested naming below)?

```
],
  "x-ms-enum": {
  "name": "OdataMetadataFormat",
  "modelAsString": true
}
```